config/nrfconnect/chip-module/Kconfig

```
@@ -146,6 +146,28 @@ config CHIP_FACTORY_DATA_USE_DEFAULT_CERTS
 	  instead of generating new ones.
 	  If this config is set to `n` new certificates will be generated.
 
+choice CHIP_FACTORY_DATA_USE_CERTS_CHOICE
+	prompt "Certificate source"
+	depends on !CHIP_FACTORY_DATA_USE_DEFAULT_CERTS
+	default CHIP_FACTORY_DATA_USE_GENERATED_CERTS
+
+	config CHIP_FACTORY_DATA_USE_GENERATED_CERTS
+		bool "generated"
+	config CHIP_FACTORY_DATA_USE_HDD_CERTS
+		bool "on HDD"
+endchoice
```

```suggestion
choice CHIP_FACTORY_DATA_CERT_SOURCE
	prompt "Attestation certificate file source"
	default CHIP_FACTORY_DATA_CERT_SOURCE_GENERATED

	config CHIP_FACTORY_DATA_CERT_SOURCE_DEVELOPMENT
		bool "Use pre-generated development certificates"
		help
	            Use pre-generated certificate files from the credentials/development/attestation/
	            directory that match the configured Product ID.
	config CHIP_FACTORY_DATA_CERT_SOURCE_GENERATED
		bool "Auto-generate certificates"
	config CHIP_FACTORY_DATA_CERT_SOURCE_USER
		bool "Use user-provided certificate files"
endchoice
```

That is, please include `CHIP_FACTORY_DATA_USE_DEFAULT_CERTS` as one of the choices. Other suggestions are just nits to make the prompts more verbose. Help texts to the remaining choices would be nice, too :)

---

I renamed it and added some more help strings. I decided to stick with `CHIP_FACTORY_DATA_USE_CERTS_CHOICE` as the name for the first option to not break the configuration intentionally :)